### PR TITLE
New package: Freenet.Freenet version 1497

### PR DESCRIPTION
**Pull request by Exorcism0666**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/136683)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978335226 2024-01-26 15:29:33.101 [CLI ] ShellExecute installer failed: 1 2024-01-26 15:29:32.957   Failed to proceed to next wizard page; aborting. 2024-01-26 15:29:32.957   Got EAbort exception.

(Automated response - build 661.)

**Comment by stephengillie** [comment]
Manual Validation ended with the same error, so I ran the installer manually and found a dependency:
image

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
I added the waiver in error. I should have added the dependency.

File: manifests/f/Freenet/Freenet/1497/Freenet.Freenet.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Freenet.Freenet
+PackageVersion: "1497"
+InstallerType: inno
+Installers:
```
Comment (stephengillie) [inline manifests/f/Freenet/Freenet/1497/Freenet.Freenet.installer.yaml:7]:
```suggestion
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Oracle.JavaRuntimeEnvironment
Installers:
```

(Automated response - build 669.)

File: manifests/f/Freenet/Freenet/1497/Freenet.Freenet.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Freenet.Freenet
+PackageVersion: "1497"
+PackageLocale: en-US
+Publisher: freenetproject.org
```
Comment (stephengillie) [inline manifests/f/Freenet/Freenet/1497/Freenet.Freenet.locale.en-US.yaml:7]:
```suggestion
Publisher: https://www.hyphanet.org/index.html
```
This URL redirects for me. Do you get the same? Should we link directly to the redirect?

File: manifests/f/Freenet/Freenet/1497/Freenet.Freenet.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Freenet.Freenet
+PackageVersion: "1497"
+PackageLocale: en-US
+Publisher: freenetproject.org
+PackageName: Freenet
+License: Proprietary
```
Comment (stephengillie) [inline manifests/f/Freenet/Freenet/1497/Freenet.Freenet.locale.en-US.yaml:9]:
```suggestion
PackageUrl: https://www.hyphanet.org/pages/download.html#windows
License: Proprietary
```

**Comment by stephengillie** [comment]
Hi @Exorcism0666,

Should this package be renamed to `Hyphanet.Freenet` or maybe `Hyphanet.Hyphanet`? 

From the [InstallerUrl redirect](https://www.hyphanet.org/index.html): 

Hyphanet website - Hyphanet is a peer-to-peer platform for censorship-resistant and privacy-respecting publishing and communication.
 The original Freenet.
 Looking for Locutus? It is now named Freenet; the team of the original Freenet continues development as Hyphanet.

Or should the `InstallerUrl` be changed to https://freenet.org/? 

It looks like the project had a fork, and your URLs point at the Hyphanet branch, while the `PackageIdentifier` and other details point at the Freenet (formerly Locutus) branch.

**Pull request closed without merging**